src/planscape/forsys/rank_projects_for_a_single_scenario.R

```diff
@@ -0,0 +1,44 @@
+suppressMessages({
+  library(dplyr)
+  library(forsys)
+  library(sf)
+})
+
+rank_projects_for_a_single_scenario <- function(input_stand_data,
+                                                priorities,
+                                                priority_weights,
+                                                stand_id_field,
+                                                proj_id_field,
+                                                stand_area_field,
+                                                stand_cost_field) {
+
+  scenario_output_fields <- c(priorities, stand_area_field, stand_cost_field)
+
+  priorities_pcp <- paste(priorities, "_PCP", sep="")
```

Comment:
can we use a more descriptive var name, specifically the pcp portion?

Reply:
Done.